inc/exhibits-detail.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * The sub-template used by page-exhibits-home.php, page-exhibits-feed.php,
+ * category-rotch-library.php and category-maihaugen-gallery.php for
+ * displaying Current Exhibits.
+ *
+ * @package MIT_Libraries_Child
+ * @since 2.0.0
+ */
+
+?>
+
+
+			<div class="exhibits-feed">
+				<div class="entry-categories">
+					<div class="entry-cats-list">
+
+							<a href="<?php echo esc_url( $cat_link ); ?>"><span class="category-bg"><span class="category-init"><?php echo esc_attr( substr( get_field( 'location' ),0,1 ) ); ?></span></span><span class="cat-name"><?php the_field( 'location' ) ?> Exhibit</span></a>
```

Comment:
Hi Thanks for the catch, I am fixing. I also noticed that the breadcrumbs are not correct , they should have the location name when displaying a particular location exhibit, should we address this now?